src/pages/additional-features/simple-scripts.mdx

```diff
@@ -0,0 +1,7 @@
+# Simple Scripts
+
+Atlas supports simple scripts, also known as native and timelock scripts. The functionality of simple scripts is explained in the ledger design document of Shelley & Mary era [here](https://github.com/IntersectMBO/cardano-ledger) and also in [cardano-node-wiki](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/reference/simple-scripts.md).
+
+Simple scripts can be specified via exposed constructors of [`GYSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#t:GYSimpleScript). They can also be read from their JSON representation as `GYSimpleScript` has a [`FromJSON`](https://hackage.haskell.org/package/aeson-2.2.3.0/docs/Data-Aeson.html#t:FromJSON) instance. We also provide a utility function, [`readSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#v:readSimpleScript), to read JSON representation from a file.
```

Comment:
```suggestion
Simple scripts can be specified via exposed constructors of the `GYSimpleScript` data type, which represents native and timelock scripts in Atlas. This data type can be used for both creating and interacting with simple scripts programmatically. 

You can specify simple scripts in their JSON representation, which is useful for reading and writing scripts from external files. The `GYSimpleScript` data type implements the [`FromJSON`](https://hackage.haskell.org/package/aeson-2.2.3.0/docs/Data-Aeson.html#t:FromJSON) typeclass, allowing for easy conversion from JSON. Additionally, we provide the [`readSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#v:readSimpleScript) utility function to load a script from a JSON file, simplifying the process of working with pre-defined scripts.
```

Reply:
This suggestion, "...type, which represents native and timelock scripts in Atlas. This data type can be used for both creating and interacting with simple scripts programmatically. " is unnecessary.